assets/map_style.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "featureType": "poi",
+    "stylers": [
+      {
+        "visibility": "off"
+      }
+    ]
+  }
+]
```

Comment:
Please organize these files in a `map/` folder

Reply:
At the end is an asset, makes no sense do a complete folder 2 files

Reply:
Keep in mind that the project will grow, and the assets folder might be filled with a lot of files.